performant-software/performant-web
# Created a projects collection with four projects from website

## What does this PR do?
- Configured a new "projects" collection in Jekyll
- Created a new layout to manage "projects" content
- Created "projects" directory
 -Save every project as ".MD" in the "_projects" collection folder
-Formatted and uploaded project images
-This closes issue #33 

## How to test?
-Front-end: navigate to the projects section and click on each project. At the moment images are placeholders and text needs revision.

-Back-end:  under "_projects/" create new "project-name.MD" file and the new project should show in the project index and project page.
